### ISSUE TITLE
Add regularization

### ISSUE THREAD

**Issue by LizAryslanova** [NuskaGirru/instrument_classifier#4]
How to add it to CNN?

**Comment by LizAryslanova** [comment]
isn't supported by ups yet, so better use dropout method
self.dropout = nn.Dropout(probability of being zeroed out)

def forward (self, x):
    x = ....
    x = self.dropout(x)

# can be used on multiple layers
